src/main/java/seedu/address/commons/util/DateUtil.java

```diff
@@ -68,4 +68,19 @@ public static int getNumOfDaysBetween(LocalDate startDate, LocalDate endDate) {
 
         return (int) startDate.until(endDate, ChronoUnit.DAYS);
     }
+
+    /**
+     * Formats the date to be printed on the loan slip.
+     *
+     * @param date date to be formatted.
+     * @return a formatted date as a {@code String}
+     */
+    public static String formatDate(String date) {
```

Comment:
Maybe it can just take in LocalDate then you don't have to getDueDate() then toString() to pass in here. Just a personal pref, up to you :^)

Reply:
Good suggestion!